### PR TITLE
Update DBX tutorial

### PR DESCRIPTION
**Pull request by TJaniF**
Updating since the modules got integrated with the OSS DBX provider :)

### CODE REVIEW

File: learn/airflow-databricks.md [hunk 1]
```diff
@@ -12,9 +12,9 @@ import databricks_tutorial_dag from '!!raw-loader!../code-samples/dags/airflow-d
...
 
-The open source [Astro Databricks provider](https://github.com/astronomer/astro-provider-databricks) provides full observability and control from Airflow so you can manage your Workflows from one place, which enables you to orchestrate your Databricks notebooks from Airflow and execute them as Databricks Workflows.
+The open source [Airflow Databricks provider](https://airflow.apache.org/docs/apache-airflow-providers-databricks/stable/index.html) provides full observability and control from Airflow so you can manage Databricks from one place, including enabling you to orchestrate your Databricks notebooks from Airflow and execute them as [Databricks Workflows jobs](https://docs.databricks.com/en/workflows/index.html).
 
-You can create a Databricks Workflow from existing Databricks notebooks as a task group in your Airflow DAG with just a few lines of code:
+You can create a Databricks jobs from existing Databricks notebooks as a task group in your Airflow DAG with just a few lines of code:
 
 ```python
```
Comment (TJaniF) [inline learn/airflow-databricks.md:19]: I think we could also remove this code block, I had it here originally thinking of it as sort of a quick start for people familiar with Airflow, but it was brought up before that it was confusing (in the dbt guide).

File: learn/airflow-databricks.md [hunk 2]
```diff
@@ -41,15 +41,11 @@ with task_group:
...
 
-
-This tutorial shows how to use the Astro Databricks provider to run two Databricks notebooks as a Databricks Workflow. If you don't use Databricks Workflows, see [Alternative ways to run Databricks with Airflow](#alternative-ways-to-run-databricks-with-airflow).
-
 :::tip Other ways to learn
 
 There are multiple resources for learning about this topic. See also:
 
-- Webinar: [How to Orchestrate Databricks Jobs Using Airflow](https://www.astronomer.io/events/webinars/how-to-orchestrate-databricks-jobs-using-airflow/).
-- Use case: [ELT with Airflow and Databricks](use-case-airflow-databricks.md) including a ready-to-use [example project repository](https://github.com/astronomer/astro-provider-databricks/blob/main/quickstart/astro-cli.md).
```
Comment (TJaniF) [inline learn/airflow-databricks.md:52]: Removing the use case because it is not updated yet.

File: learn/airflow-databricks.md [hunk 3]
```diff
@@ -58,11 +54,11 @@ There are multiple resources for learning about this topic. See also:
...
-- [Inject parameters](#parameters) into your Databricks Workflow at the Workflow-level. These parameters can be dynamic and retrieved at runtime from other Airflow tasks.
+- Use [task groups](task-groups.md) within Databricks jobs, enabling you to collapse and expand parts of larger Databricks jobs visually.
+- Leverage [Airflow datasets](airflow-datasets.md) to trigger Databricks jobs from tasks in other DAGs in your Airflow environment or using the Airflow REST API [Create dataset event endpoint](https://airflow.apache.org/docs/apache-airflow/stable/stable-rest-api-ref.html#operation/create_dataset_event), allowing for a data-driven architecture.
+- Use familiar Airflow code as your interface to orchestrate Databricks notebooks as jobs.
+- [Inject parameters](#parameters) into your Databricks job at the job-level. These parameters can be dynamic and retrieved at runtime from other Airflow tasks.
+- Provider version 6.8.0+: Repair single tasks in your Databricks job from the Airflow UI. If a task fails, you can [re-run it](#repairing-a-databricks-job) using an [operator extra link](operator-extra-link-tutorial.md) in the Airflow UI.
```
Comment (TJaniF) [inline learn/airflow-databricks.md:61]: This is now also possible in the DBX UI, but still nice to be able to do from within Airflow.

File: learn/airflow-databricks.md [hunk 4]
```diff
@@ -134,65 +124,75 @@ You can orchestrate any Databricks notebooks in a Databricks Workflow using the
...
+        notebook_params={
+            "my_date": "{{ ds }}"
+        },
+    )
+    ```
+    
+    To retrieve this parameter inside your Databricks notebook add the following code to a Databricks notebook cell:
+
+    ```python
+    dbutils.widgets.text("my_date", "my_default_value", "Description")
```
Comment (TJaniF) [inline learn/airflow-databricks.md:176]: This took me about 30min to figure out, so I think it is worth having it here 😅

File: learn/airflow-databricks.md [hunk 5]
```diff
@@ -134,65 +124,75 @@ You can orchestrate any Databricks notebooks in a Databricks Workflow using the
...
 
-The Astro Databricks provider includes functionality to repair a failed Databricks Workflow by making a repair request to the [Databricks Jobs API](https://docs.databricks.com/api-explorer/workspace/jobs/repairrun). Databricks expects a single repair request for all tasks that need to be rerun in one cluster, this can be achieved via the Airflow UI by using the operator extra link **Repair All Failed Tasks**. If you would be using Airflow's built in [retry functionality](rerunning-dags.md#automatically-retry-tasks) a separete cluster would be created for each failed task.
+The Airflow Databricks provider version 6.8.0+ includes functionality to repair a failed Databricks job by making a repair request to the [Databricks jobs API](https://docs.databricks.com/api-explorer/workspace/jobs/repairrun). Databricks expects a single repair request for all tasks that need to be rerun in one cluster, this can be achieved via the Airflow UI by using the operator extra link **Repair All Failed Tasks**. If you would be using Airflow's built in [retry functionality](rerunning-dags.md#automatically-retry-tasks) a separate cluster would be created for each failed task.
```
Comment (TJaniF) [inline learn/airflow-databricks.md:192]: 6.8.0 should be released with the next provider wave.

File: learn/airflow-databricks.md (hunk 1 above)
Comment (kentdanas) [reply learn/airflow-databricks.md:19]: yeah let's remove it

File: learn/airflow-databricks.md [hunk 6]
```diff
@@ -12,9 +12,9 @@ import databricks_tutorial_dag from '!!raw-loader!../code-samples/dags/airflow-d
 
 [Databricks](https://databricks.com/) is a popular unified data and analytics platform built around [Apache Spark](https://spark.apache.org/) that provides users with fully managed Apache Spark clusters and interactive workspaces.
 
-The open source [Astro Databricks provider](https://github.com/astronomer/astro-provider-databricks) provides full observability and control from Airflow so you can manage your Workflows from one place, which enables you to orchestrate your Databricks notebooks from Airflow and execute them as Databricks Workflows.
+The open source [Airflow Databricks provider](https://airflow.apache.org/docs/apache-airflow-providers-databricks/stable/index.html) provides full observability and control from Airflow so you can manage Databricks from one place, including enabling you to orchestrate your Databricks notebooks from Airflow and execute them as [Databricks Workflows jobs](https://docs.databricks.com/en/workflows/index.html).
```
Comment (kentdanas) [inline learn/airflow-databricks.md:15]:
```suggestion
The open source [Airflow Databricks provider](https://airflow.apache.org/docs/apache-airflow-providers-databricks/stable/index.html) provides full observability and control from Airflow so you can manage Databricks from one place, including enabling you to orchestrate your Databricks notebooks from Airflow and execute them as [Databricks jobs](https://docs.databricks.com/en/workflows/index.html).
```
Assume we should remove "Workflow" here?

File: learn/airflow-databricks.md (hunk 3 above)
Comment (kentdanas) [inline learn/airflow-databricks.md:61]:
```suggestion
- Repair single tasks in your Databricks job from the Airflow UI (Provider version 6.8.0+ is required). If a task fails, you can [re-run it](#repairing-a-databricks-job) using an [operator extra link](operator-extra-link-tutorial.md) in the Airflow UI.
```

**Review by kentdanas — APPROVED** [review]
Looks great, just a few minor things

**Comment by TJaniF** [comment]
Thank you @kentdanas ! 
@lzdanski ready for you :)

**Comment by kentdanas** [comment]
Since there wasn't much copy changed in this, I'm going to go ahead and merge it and save some review bandwidth

**Pull request merged**